File: 05. ConditionalStatements/03. BiggestOfThree/03. BiggestOfThree.cs
using System;

class BiggestOfThree
{
    //Write a program that finds the biggest of three integers using nested if statements.

    static void Main(string[] args)
    {
        Console.Write("Enter a number: ");
        int numberOne = int.Parse(Console.ReadLine());
        Console.Write("Enter a second number: ");
        int numberTwo = int.Parse(Console.ReadLine());
        Console.Write("Enter a third number: ");
        int numberThree = int.Parse(Console.ReadLine());

        if ((numberOne >= numberTwo && numberOne >= numberThree))
        {
            Console.WriteLine("{0} is the biggest number.", numberOne);
        }
        else if (numberTwo >= numberOne && numberTwo >= numberThree)
        {
            Console.WriteLine("{0} is the biggest number.", numberTwo);
        }
        else
        {
            Console.WriteLine("{0} is the biggest number.", numberThree);
        }


    }
}

File: JavaScript/JS Fundamentals/Homework/03.ConditionalStatements/03.ConditionalStatements/03.BiggestOfThree.js
// Write a script that finds the biggest of
// three integers using nested if statements.

var num1 = 1;
var num2 = 2;
var num3 = 3;

if (num1 > num2 && num1 > num3) {
    console.log('Biggest number is: ' + num1)
} else if (num2 > num1 && num2 > num3) {
    console.log('Biggest number is: ' + num2)
} else if (num3 > num1 && num3 > num2) {
    console.log('Biggest number is: ' + num3)
}

